### PR TITLE
chore: Runs the tests with Spec for clearer output

### PR DESCRIPTION
**Pull request by spydon**
# Description

Running the tests with [Spec](https://github.com/invertase/spec) gives insanely much clearer output.
(Checkout the test check on this PR)

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples`.

## Breaking Change

- [x] No, this is *not* a breaking change.

## Related Issues

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flame Style Guide]: https://github.com/flame-engine/flame/blob/main/STYLEGUIDE.md
[Conventional Commit]: https://conventionalcommits.org

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -48,5 +48,5 @@ jobs:
         with:
           channel: 'stable'
       - uses: bluefireteam/melos-action@main
-      - run: melos run test
+      - uses: bluefireteam/spec-action@main
```
Comment (erickzanardo) [inline .github/workflows/cicd.yml:51]: Oh, is this new?

Comment (spydon) [reply .github/workflows/cicd.yml:51]: Yep, created it as an experiment like half an hour ago :smile:

Comment (spydon) [reply .github/workflows/cicd.yml:51]: (it's pretty much exactly the same as the melos-action)

Comment (erickzanardo) [reply .github/workflows/cicd.yml:51]: pretty cool, we should make a tweet

Comment (spydon) [reply .github/workflows/cicd.yml:51]: Good idea!
It's here: https://github.com/marketplace/actions/spec-action

**Pull request merged**